doc/src/tutorials/creating-packages.md

```diff
@@ -605,9 +605,20 @@ may fit your package better.
 
      * `CPLEX.jl` wraps the `CPLEX` library, which can be identified easily in a web search.
      * `MATLAB.jl` provides an interface to call the MATLAB engine from within Julia.
+
 7. Avoid naming a package closely to an existing package
      * `Websocket` is too close to `WebSockets` and can be confusing to users. Rather use a new name such as `SimpleWebsockets`.
 
+8. Avoid using a distinctive name that is already in use in another unrelated project.
```

Comment:
```suggestion
8. Avoid using a distinctive name that is already in use in a well known, unrelated project.
```
As Sundar R noted on Zulip, without a "well known" or "popular" clause, names like Oxygen.jl could be suspect as that name is both distinctive and in use elsewhere. However none of the other usages are particularly notable.